layouts/_default/baseof.html

```diff
@@ -5,15 +5,15 @@
 		<header>
 {{ partial "header.html" . }}
 {{ if .Site.Params.NavigationInHeader }}
-		<nav>
-{{ partial "menu.html" .Site.Menus.main }}
+		<nav class="nav-in-header">
```

Comment:
Is there any reason for this class? I think we should try to keep the HTML as clean as possible and one can always use the following selectors:
```css
body > nav { }
body > header > nav { }
```